ROADMAP.md

```diff
@@ -12,56 +12,57 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
-  * Isolate and parallelize storage provider testing [#1218](https://github.com/rook/rook/issues/1218)
-  * Longhaul testing pipeline [#1847](https://github.com/rook/rook/issues/1847)
-* Custom resource validation, progress, status [#1539](https://github.com/rook/rook/issues/1539)
-* Design for Volume Snapshotting and policies (consider aligning with SIG-storage) [#1552](https://github.com/rook/rook/issues/1552)
-* Support for dynamic provisioning of new storage types
-  * Dynamic bucket provisioning [#1705](https://github.com/rook/rook/issues/1705)
-  * Dynamic database provisioning [1704](https://github.com/rook/rook/issues/1704)
 * New storage providers
   * NFS operator and CRDs (backed by arbitrary PVs) [#1551](https://github.com/rook/rook/issues/1551)
-  * Cassandra design [#1910](https://github.com/rook/rook/issues/1910)
-* CockroachDB
-  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
-  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
-  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
-* Minio
-  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
-  * Run on arbitrary PVs [#919](https://github.com/rook/rook/issues/919)
+  * Cassandra operator [#1910](https://github.com/rook/rook/issues/1910)
+  * Nexenta EdgeFS operator [#1532](https://github.com/rook/rook/issues/1532)
 * Ceph
+  * Declare Ceph CRDs to be stable v1
   * Added support for Mimic and later versions in addition to Luminous [#2004](https://github.com/rook/rook/issues/2004)
   * Automated upgrade support (initial) [#997](https://github.com/rook/rook/issues/997)
-  * Manage an existing Ceph cluster (basic) [#1868](https://github.com/rook/rook/issues/1868)
+  * CSI plug-in documentation [#2234](https://github.com/rook/rook/pull/2234)
   * OSDs
     * Run on arbitrary PVs (local storage) as an alternative to host path [#796](https://github.com/rook/rook/issues/796) [#919](https://github.com/rook/rook/issues/919)
-    * Minimize or eliminate running with privileged containers (e.g., reduced access to /dev) [#1944](https://github.com/rook/rook/issues/1944)
-    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
     * ceph-volume is used for provisioning [#1342](https://github.com/rook/rook/issues/1342)
   * Mgr and plugins
-    * Placement group balancer support (enable the mgr module)
-  * File
-    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
-    * Dynamic Volume Provisioning for CephFS [#1125](https://github.com/rook/rook/issues/1125)
+    * Enable the Mimic dashboard with a self-signed cert and generated admin creds
+    * Enable the orchestrator modules for nautilus
   * Object
-    * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
     * CRD for object store users [#1583](https://github.com/rook/rook/issues/1583)
 
 ## Rook 1.0
```

Comment:
we can probably add some Cassandra planned items to v1.0 also, see https://github.com/rook/rook/issues/2294#issuecomment-442057097

Reply:
@yanniszark what would you like to add? It could also be added with a later PR since the purpose of this was more about clarifying 0.9.